wrapper.html

```diff
@@ -166,46 +175,42 @@
 	<link rel='stylesheet' id='baseline-rtl-css' href='https://dbqvwi2zcv14h.cloudfront.net/ak/ak-v3-rtl.css'
 		type='text/css' media='all' />
 	{% endif %}
-
+	<!-- If custom WebP URL is set, use WebP with the page_background_image as a fallback. Else just use page_background_image. -->
+	{% if page.custom_fields.page_background_image %}
 	<style>
-		{
-			% if page.custom_fields.page_background_image %
-		}
-
-		#body-mobile-background {
-			background-color: {
-					{
-					templateset.custom_fields.color_page
+		{% if page.custom_fields.page_background_image_webp_url %}
+			@supports not (background-image: url('{{ page.custom_fields.page_background_image_webp_url }}')) {

```

Comment:
@sukhada does this work? According to @support docs: (https://developer.mozilla.org/en-US/docs/Web/CSS/@supports):

> The most basic supports condition is a simple declaration (a property name followed by a value, separated by a colon). The declaration must be surrounded by parentheses. The following example returns true if the browser's transform-origin property considers 5% 5% valid:
> @supports (transform-origin: 5% 5%) {}

I think in this case, the rule would return valid, but the browser might not be able to display a webp format image?

We might need to use the `<picture>` tag here: https://usefulangle.com/post/114/webp-image-in-html-with-fallback

Reply:
TL;DR implementing the `picture` tag as you suggested actually had a negative effect on pagespeed, and as you pointed out, the `@supports` tag didn't actually work. I'm removing the webP implementation for now, because it actually doesn't seem to affect pagespeed that much